packages/shared/src/components/squads/Details.tsx

```diff
@@ -155,85 +151,126 @@ export function SquadDetails({
-          <div className="flex flex-col">
-            <h4 className="mb-2 typo-headline">Invitation permissions</h4>
-            <p className="mb-4 text-theme-label-tertiary typo-callout">
-              Choose who is allowed to invite new members to this squad.
-              {createMode && ' You can always change this setting later.'}
-            </p>
-            <Radio
-              name="memberInviteRole"
-              options={memberRoleOptions}
-              value={memberInviteRole}
-              onChange={(value) =>
-                setMemberInviteRole(value as SourceMemberRole)
-              }
+            <TextField
+              label="Squad handle"
+              inputId="handle"
+              hint={handleHint}
+              valid={!handleHint}
+              name="handle"
+              leftIcon={<AtIcon />}
+              value={activeHandle ?? ''}
+              onChange={() => !!handleHint && setHandleHint(null)}
+              className={{
+                hint: 'text-theme-status-error',
+                container: classNames('w-full', !handleHint && 'mb-1'),
+              }}
             />
-          </div>
+            {!isDescriptionOpen && (
+              <button
+                className="mr-auto typo-callout text-theme-label-tertiary"
+                type="button"
+                onClick={() => {
+                  setDescriptionOpen((current) => !current);
+                }}
+              >
+                + Add description
+              </button>
+            )}
+            {isDescriptionOpen && (
+              <Textarea
+                label="Squad description"
+                inputId="description"
+                name="description"
+                hint="(optional)"
+                rows={4}
+                value={description ?? ''}
+                maxLength={250}
+                className={{
+                  hint: '-mt-8 py-2 pl-4',
+                  container: 'w-full',
+                }}
+              />
+            )}
+            <h4 className="mt-2 font-bold typo-body">
+              {createMode ? 'Squad type' : 'Squad'}
+            </h4>
+            <div className="flex flex-row gap-4 rounded-16 border-2 border-theme-divider-tertiary">
+              <SquadTypeCard
+                title="Private squad"
+                description="Only people who join the squad can see the content"
+                isSelected={!isPublic}
+              />
+              <SquadTypeCard
+                title={
+                  <div className="flex gap-2 items-center">
+                    Public squad <BetaBadge />
+                  </div>
+                }
+                description="Everyone can see the content and the posts may appear on the main feed"
+                isSelected={isPublic}
+                buttonProps={{
+                  text: 'Join waitlist',
+                  tag: 'a',
+                  // @ts-expect-error Property 'disabled' does not exist on type
```

Comment:
We are not even referring disabled here, does it expect it?
Maybe we better off fixing the issues down the props.

Reply:
`target` would be an error here `disabled` is just a typo, `@ts-expect-error` disables the next error no matter what but requires some kind of note. 

<img width="802" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/ac1c8e9a-f526-4510-87f9-bc61ebddc350">